book/release-service.md

```diff
@@ -39,8 +28,95 @@ The diagram below shows the flow of custom resources and the orchestration of pi
 
 ![](../diagrams/release-service/hacbs-release-service-data-flow.jpg)
 
+## Terminology
+
+**Snapshot** -  The custom resource that contains the list of all Components of an Application with their Component Image digests. Once created, the list of Components with their images is immutable. The Integration service updates the status of the resource to reflect the testing outcome.
+**Release Pipeline** -  An instance of a Pipeline that is orchestrated by the Release Service.
+**Bundle** - A tekton bundle (image) that contains Tekton resources
+**Development Workspace** - workspace shared by 1 or more team members when code is built and tested.
+**Managed Workspace** - workspace that is managed by an SRE or Release Engineering team that is controlled. This is where Release pipeline execute.
+
+## Resources
+Below are the list of CRs that the Release service is responsible for interacting and performing operations:
+
+### CREATE
+
+| Custom Resource            | When?                                                    | Why?                                                                                       |
+|----------------------------|----------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| SnapshotEnvironmentBinding | When deploying the Snapshot to an Environment            | Create a SnapshotEnvironmentBinding for the Referenced Environment when one does not exist |
+| PipelineRuns               | Once a Release has been composed and ready for execution | To perform the steps in the Release Strategy                                               |
+
+### READ
+
+| Custom Resources        | When?                                                  | Why?                                                                     |
+|-------------------------|--------------------------------------------------------|--------------------------------------------------------------------------|
+| Application & Component | When validating ReleasePlans and ReleasePlanAdmissions | To ensure data consistency                                               |
+| ReleasePlan             | When starting to process a Release                     | To determine the target workspace to run the Release Pipeline            |
+| ReleasePlanAdmission    | When starting to process a Release                     | To determine the Release Strategy to use to compose the Release Pipeline |
```

Comment:
for consistency, probably no space in Release Strategy. It is a CRD name

Reply:
fixed